misk/testing/src/main/kotlin/misk/testing/MiskTestExtension.kt

```diff
@@ -0,0 +1,48 @@
+package misk.testing
+
+import com.google.inject.Guice
+import com.google.inject.Module
+import com.sun.org.apache.xpath.internal.operations.Mod
+import misk.inject.uninject
+import org.junit.jupiter.api.extension.*
+import org.junit.platform.commons.util.AnnotationUtils
+import kotlin.reflect.full.createInstance
+
+internal class MiskTestExtension: BeforeEachCallback, AfterEachCallback {
+    override fun beforeEach(context: ExtensionContext?) {
```

Comment:
You Might be able to make `context` non-nullable here, sometimes Intellij defaults to assuming something is nullable.